lib/vnet/service_windows.go

```diff
@@ -163,7 +163,7 @@ loop:
 
 func (s *windowsService) run(ctx context.Context, args []string) error {
 	var cfg windowsAdminProcessConfig
-	app := kingpin.New(serviceName, "Teleport VNet Windows Service")
+	app := kingpin.New("TeleportVNet", "Teleport VNet Windows Service")
```

Comment:
What's the reason for this change?

Reply:
uh no reason I'll revert it. This string actually doesn't matter at all it just got changed back and forth when i deleted the serviceName constant to make it per user and then reverted that